app/src/main/java/org/simple/clinic/enterotp/EnterOtpEffectHandler.kt

```diff
@@ -50,6 +50,7 @@ class EnterOtpEffectHandler @AssistedInject constructor(
         .addConsumer(BlockOtpEntryUntil::class.java, { uiActions.showOtpEntryMode(BruteForceOtpEntryLocked(it.blockTill)) }, schedulers.ui())
         .addAction(HideErrors::class.java, { uiActions.hideError() }, schedulers.ui())
         .addConsumer(ShowIncorrectOtpLimitReachedError::class.java, { uiActions.showLimitReachedError(it.attemptsMade) }, schedulers.ui())
+        .addConsumer(ShowIncorrectOtpError::class.java, { uiActions.showIncorrectOtpErrorAttempt(it.attemptsMade, it.attemptsRemaining) }, schedulers.ui())
```

Comment:
Same comment as before, is there a reason why we are using UI actions for everything? 😅